DemoApp/DemoApp/Views/TMCardPage.xaml

```diff
@@ -92,7 +93,7 @@
           <StackLayout BackgroundColor="Transparent"
                        HorizontalOptions="End"
                        Padding="0,10,0,10">
-            <local:TMButton Title="Button"></local:TMButton>
+            <Button:TMButton Title="Button" Size="Default"></Button:TMButton>

```

Comment:
Should be `Modus:TMButton`. All the controls should be exposed under the Trimble.Modus.Components namespace